index.bs

```diff
@@ -279,5 +382,56 @@ Threat model {#security-threat}
 Mitigations and defenses {#security-defenses}
 ---------------------------------------------
 
+Appending A: Messages
+=====================
+The following messages are defined with [CDDL](https://datatracker.ietf.org/doc/draft-ietf-cbor-cddl/). When integer keys are used, a comment is appended to the line to indicate the name of the field. Each root message (one that can be put into a QUIC stream without being inclosed by another message) has a comment indicating the message type key.
```

Comment:
Suggest line breaks here and below (I would use GitHub's suggest here as well but there's no column count in the comment editor).

Reply:
Done